student_explorer/cron.py

```diff
@@ -1,8 +1,6 @@
-# -*- coding: future_fstrings -*-
+ #python 3 support
 
-from __future__ import print_function #python 3 support
-
-from __future__ import absolute_import #py3 management conflict in cron
+ #py3 management conflict in cron
```

Comment:
What does this comment mean? Will cron run in Python 3 environment?

Reply:
Looks like the absolute_import was removed but the comment remained. I can remove this comment. There was a problem with python2 running cron before.